### PR TITLE
Add README.md to dummy perception publisher

### PR DESCRIPTION
**Pull request by YoheiMishina**
## Description

Add new README.md in dummy perception publisher

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [pull request guidelines][pull-request-guidelines]
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: simulator/util/dummy_perception_publisher/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# dummy_perception_publisher
+
+## Purpose
+
+This node publishers the result of the dummy detection with the type of perception. if `real_use_param_` is True, the data containing the Feature is delivered, and if False, the data without the Feature is delivered.
```
Comment (kmiya) [inline simulator/util/dummy_perception_publisher/README.md:5]:
```suggestion
This node publishes the result of the dummy detection with the type of perception. If `real_use_param_` is True, the data containing the Feature is delivered, and if False, the data without the Feature is delivered.
```
Nit

File: simulator/util/dummy_perception_publisher/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+| Name                     | Type                                       | Description    |
+| ------------------------ | ------------------------------------------ | -------------- |
+| `/tf`                    | `tf2_msgs/TFMessage`                       | TF (self-pose) |
+| `input/object`           | `dummy_perception_publisher::msg::Object`  | dummy detection objects |
+
+### Output
+
+| Name         | Type                                       | Description                                                                                           |
+| ------------ | ------------------------------------------ | ----------------------------------------------------------------------------------------------------- |
+| `output/dynamic_object` | `autoware_perception_msgs::msg::DetectedObjectsWithFeature / autoware_auto_perception_msgs::msg::DetectedObjects` | Publishers objects.(True:w/Feature, False:wo/Feature) |
```
Comment (kmiya) [inline simulator/util/dummy_perception_publisher/README.md:22]:
```suggestion
| `output/dynamic_object` | `autoware_perception_msgs::msg::DetectedObjectsWithFeature / autoware_auto_perception_msgs::msg::DetectedObjects` | Publishes objects (True: w/ Feature, False: w/o Feature) |
```
Nit

File: simulator/util/dummy_perception_publisher/README.md (hunk 1 above)
Comment (YoheiMishina) [reply simulator/util/dummy_perception_publisher/README.md]: fix at 6d8ec832f2bdc0ec61e3b30f0af5e205006e60dc

File: simulator/util/dummy_perception_publisher/README.md (hunk 2 above)
Comment (YoheiMishina) [reply simulator/util/dummy_perception_publisher/README.md]: fix at 6d8ec832f2bdc0ec61e3b30f0af5e205006e60dc

File: simulator/util/dummy_perception_publisher/README.md (hunk 1 above)
Comment (kmiya) [reply simulator/util/dummy_perception_publisher/README.md]: `perception. if` is still lower-case.

File: simulator/util/dummy_perception_publisher/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Parameters
+
+| Name                        | Type   | Default Value | Explanation                                                                                        |
+| --------------------------- | ------ | ------------- | -------------------------------------------------------------------------------------------------- |
+| `visible_range`             | double | 100.0         | sensor visible range [m]                                                                           |
+| `detection_successful_rate` | double | 0.8           | sensor detection rate. (min) 0.0 - 1.0(max)                                                        |
+| `enable_ray_tracing`        | bool   | true          | if True, use ray tracking                                                                          |
+| `use_object_recognition`    | bool   | true          | if True, publish objects topic                                                                     |
+| `real_use_param_`           | bool   | true          | This parameter is a switch that toggles the presence or absence of "feature" in the output object. |
```
[the remark this answers is not in the document]
Comment (kenji-miyake) [reply simulator/util/dummy_perception_publisher/README.md:33]: Also, `use_real_param` is more natual? :thinking:

[the remark this answers is not in the document]
Comment (YoheiMishina) [reply simulator/util/dummy_perception_publisher/README.md]: Is it correct? 10ee0bfab24ef52391b0323eacb4f72613d836c1

File: simulator/util/dummy_perception_publisher/README.md (hunk 1 above)
Comment (YoheiMishina) [reply simulator/util/dummy_perception_publisher/README.md]: Sorry, I fix completely in 5ba71acecf69c3781fe9be463c0a7fe2a7c3c88b

File: simulator/util/dummy_perception_publisher/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kenji-miyake) [reply simulator/util/dummy_perception_publisher/README.md]: No, please update .cpp as well.
@tkimura4 @taikitanaka3 Could you talk to Mishina-san, please? :pray:

File: simulator/util/dummy_perception_publisher/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# dummy_perception_publisher
+
+## Purpose
+
+This node publishes the result of the dummy detection with the type of perception. f `real_use_param_` is True, the data containing the Feature is delivered, and if False, the data without the Feature is delivered.
```
Comment (kmiya) [inline simulator/util/dummy_perception_publisher/README.md:5]:
```suggestion
This node publishes the result of the dummy detection with the type of perception. If `real_use_param_` is True, the data containing the Feature is delivered, and if False, the data without the Feature is delivered.
```
Missing `I`. Please check carefully.

File: simulator/util/dummy_perception_publisher/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (YoheiMishina) [reply simulator/util/dummy_perception_publisher/README.md]: I update .cpp 2711ab9d012cb2e8d49a5ffc35ba86b1066c6f4a

File: simulator/util/dummy_perception_publisher/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+| Name           | Type                                      | Description             |
+| -------------- | ----------------------------------------- | ----------------------- |
+| `/tf`          | `tf2_msgs/TFMessage`                      | TF (self-pose)          |
+| `input/object` | `dummy_perception_publisher::msg::Object` | dummy detection objects |
+
+### Output
+
+| Name                    | Type                                                                                                              | Description                                           |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- |
+| `output/dynamic_object` | `autoware_perception_msgs::msg::DetectedObjectsWithFeature / autoware_auto_perception_msgs::msg::DetectedObjects` | Publishes objects (True:w/Feature, False:w/o Feature) |
```
[the remark this answers is not in the document]
Comment (YoheiMishina) [reply simulator/util/dummy_perception_publisher/README.md]: Thank you for telling me about it. I fixed it.f25d3bea3fbbbb78cddb075db6344377a04566ec

[the remark this answers is not in the document]
Comment (kmiya) [reply simulator/util/dummy_perception_publisher/README.md]: f25d3be

File: simulator/util/dummy_perception_publisher/README.md (hunk 4 above)
Comment (kmiya) [reply simulator/util/dummy_perception_publisher/README.md]: f25d3be

Comment (YoheiMishina) [reply simulator/util/dummy_perception_publisher/README.md]: Sorry again, I double checked it.f25d3bea3fbbbb78cddb075db6344377a04566ec

File: simulator/util/dummy_perception_publisher/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kenji-miyake) [reply simulator/util/dummy_perception_publisher/README.md]: Thank you, but one more thing.

It seems this parameter is introduced in #562.
However, I feel the naming is a bit confusing.
How about renaming it to `use_real_param`?

[the remark this answers is not in the document]
Comment (kenji-miyake) [reply simulator/util/dummy_perception_publisher/README.md]: Thank you!

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**